app/src/main/java/org/feature/fox/coffee_counter/ui/profile/ProfileView.kt

```diff
@@ -1,16 +1,326 @@
+                    IconButton(onClick = { showPassword.value = true }) {
+                        Icon(
+                            imageVector = Icons.Filled.VisibilityOff,
+                            contentDescription = stringResource(R.string.show_password)
+                        )
+                    }
+                }
+            }
+        )
+
+    }
+}
+
+@Composable
+fun AdminCheckbox(isAdmin: Boolean) {
+    val isChecked = remember { mutableStateOf(isAdmin) }
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.Center
+    ) {
+        Checkbox(
+            checked = isChecked.value,
+            onCheckedChange = { isChecked.value = it }
+        )
+        Text(text = stringResource(R.string.admin_label))
+    }
+}
+
+
+@Composable
+fun ProfileIcon() {
+    val painter = rememberAsyncImagePainter(R.drawable.ic_baseline_person_24)
+
+    Column(
+        modifier = Modifier
+            .padding(8.dp)
+            .fillMaxWidth(),
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        Image(
+            painter = painter,
+            contentDescription = stringResource(R.string.profile_image_label),
+            modifier = Modifier
+                .wrapContentSize()
+                .size(150.dp),
+            contentScale = ContentScale.Crop
+        )
+    }
+}
+
+@Composable
+fun ButtonRow() {
+    val notification = rememberSaveable() { mutableStateOf("") }
+    if (notification.value.isNotEmpty()) {
+        Toast.makeText(LocalContext.current, notification.value, Toast.LENGTH_SHORT).show()
+        notification.value = ""
+    }
+
+    Column(
+        modifier = Modifier
+            .verticalScroll(rememberScrollState())
+            .padding(8.dp)
+    ) {
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(8.dp),
+            horizontalArrangement = Arrangement.SpaceEvenly,
+        ) {
+            Card(
+                shape = CircleShape,
+                backgroundColor = CrayolaBrown,
+                modifier = Modifier
+                    .padding(8.dp)
+                    .size(50.dp)
+            ) {
+                IconButton(
+                    onClick = { notification.value = "Profile updated" },
+                ) {
+                    Icon(
+                        imageVector = Icons.Filled.SaveAlt,
+                        null,
+                        tint = Color.White,
+                        modifier = Modifier.fillMaxWidth()
+                    )
+                }
+            }
+
+            Card(
+                shape = CircleShape,
+                backgroundColor = CrayolaBrown,
+                modifier = Modifier
+                    .padding(8.dp)
+                    .size(50.dp)
+            ) {
+                IconButton(
+                    onClick = {
+                        notification.value = "Profile deleted"
```

Comment:
Strings should be moved to `strings.xml`.

Reply:
Should be resolved with 68043254c9bf398ad297900a56d12ea722f934ee